### DISCUSSION TITLE
How to identify why a property is not mapped

### DISCUSSION THREAD

**Discussion by elendil-software** [riok/mapperly#1441] [Q&A] [answered]
Hi,

I have an existing mapper that works perfectly.
Now, I have added a new property on both sides but it is not mapped.
I tried to rename the mapper to be sure it has been regenerated. I checked the type and the name of the property, they are identical on both sides. I also tried to edit the generated code to check that properties can be accessed. Everything is OK, I don't understand.
The target is a gRPC generated class and the source is a simple POCO.

Is there a way to know why a property has been ignored ?

Julien

**Comment by elendil-software** [comment]
Solved after having restarted the IDE.

Anyway it would be nice to have some log of the generation (if possible) to debug this kind of issue

**Reply by latonz** [reply, marked as answer]
If it was solved with a restart of the IDE it was likely a bug in loading / running the source generator… otherwise there should usually be RMG* diagnostics be reported if a member isn‘t mapped.
